src/main.rs

```diff
@@ -18,7 +18,12 @@ fn main() -> Result<(), Error> {
             Arg::with_name("v")
                 .short("v")
                 .multiple(true)
-                .help("Verbosity, add more for more verbose output."),
+                .help("Sets the verbosity level"),
```

Comment:
I would keep this the same otherwise it isn't clear you can set multiple

Reply:
How `Increases the verbosity level`? I feel like it's pretty standard for verbosity to work like this in command line programs. For instance this is how pytest does it:

```
-v, --verbose         Increase verbosity
```

It's also worth noting that the changes from #3 make it even more obvious by actually rendering the option like this `-v...`

Reply:
Even Cargo itself has a description stating you can add it multiple times (from `cargo --help`):

```
  -v, --verbose...          Use verbose output (-vv very verbose/build.rs
                            output)
```

Which to me implies the `...` is not clear enough, but I also agree it is fairly standard. Either way, it doesn't hurt to add a few more words here.